src/debug/di/module.cpp

```diff
@@ -3391,6 +3391,49 @@ mdSignature CordbILCode::GetLocalVarSigToken()
     return m_localVarSigToken;
 }
 
+HRESULT CordbILCode::CreateNativeBreakpoint(ICorDebugFunctionBreakpoint **ppBreakpoint)
+{
+    PUBLIC_REENTRANT_API_ENTRY(this);
+    FAIL_IF_NEUTERED(this);
+    VALIDATE_POINTER_TO_OBJECT(ppBreakpoint, ICorDebugFunctionBreakpoint **);
+
+    HRESULT hr;
+    ULONG32 size = GetSize();
+    LOG((LF_CORDB, LL_INFO10000, "CordbILCode::CreateNativeBreakpoint, size=%d, this=0x%p\n",
+        size, this));
+
+    ULONG32 offset = 0;
+    // Make sure the offset is within range of the method.
```

Comment:
This comment is misleading and should be removed. It states the offset and the result of GetSize() are either both native bytes or both IL bytes but that is not the case in the new scenario you are adding. GetSize() will be managed bytes but the 0 is a native byte offset.